README.md

```diff
@@ -1,11 +1,33 @@
-# Packetgraph
+# Packetgraph ?
 
-Packetgraph library is build upon [DPDK](http://dpdk.org/) making a collection
-of network bricks you can connect to form a network graph.
-The goal of this project is to provide a low-latency software defined
-network.
-Everyone is free to use this library to build up there own network
-infrastructure.
+Packetgraph library is build upon [DPDK](http://dpdk.org/) making a
```

Comment:
Presenting pg by saying: "Packetgraph library is build upon [DPDK]" make Packetgraph first a sub-dpdk library then a network graph library's.

So maybe we should say at first, than pg aim to create a network graph then say it's base on DPDK:
"Packetgraph is a library aiming to give the user a tool to build networks graph easily, It's build upon the well tested and fast DPDK library's"